tests/unit/charms/tls_certificates_interface/v2/test_tls_certificates_v2_provides.py

```diff
@@ -848,6 +1147,65 @@ def test_given_multiple_certificate_in_relation_data_when_get_issued_certificate
         certificates = self.harness.charm.certificates.get_issued_certificates()
         self.assertEqual(certificates, expected_certificates)
 
+    def test_given_multiple_certificate_in_relation_data_when_get_issued_certificates_and_unit_not_leader_then_returned_dicts_has_empty_certificates_list(
```

Comment:
```suggestion
    def test_given_multiple_certificate_in_relation_data_when_get_issued_certificates_and_unit_not_leader_then_returned_dict_has_empty_certificates_list(
```

But as above, the list will be empty independent from the number of certs.

Reply:
This is removed.